src/main/java/lotto/enums/Prize.java

```diff
@@ -0,0 +1,36 @@
+package lotto.enums;
+
+public enum Prize {
+    FIFTH(3, false, 5000L),
+    FOURTH(4, false, 50000L),
+    THIRD(5, false, 1500000L),
+    SECOND(5, true, 30000000L),
+    FIRST(6, false, 2000000000L);
```

Comment:
구분자를 넣어 가독성을 좋게 만들어 봅시다 🔥 
```suggestion
    FIRST(6, false, 2_000_000_000L);
```